src/GraphQL/loaders/defaultGraphQLTypes.js

```diff
@@ -1020,6 +1021,56 @@ const SIGN_UP_RESULT = new GraphQLObjectType({
   },
 });
 
+const ElEMENT = new GraphQLObjectType({
+  name: 'Element',
+  description:
+    'The SignUpResult object type is used in the users sign up mutation to return the data of the recent created user.',
+  fields: {
+    value: {
+      description: 'Return the value of the element in the array',
+      type: new GraphQLNonNull(ANY),
+    },
+  },
+});
+
+// Default static union type, we update types and resolveType function later
+const ARRAY_RESULT = new GraphQLUnionType({
+  name: 'ArrayResult',
+  description:
+    'Use Inline Fragment on Array to get results: https://graphql.org/learn/queries/#inline-fragments',
+  types: () => [ElEMENT],
+  resolveType: () => {
+    return ElEMENT;
+  },
+});
+
+const loadArrayResult = (parseGraphQLSchema, parseClasses) => {
+  const ArrayResultTypeIndex = parseGraphQLSchema.graphQLTypes.findIndex(
+    type => type.name === 'ArrayResult'
+  );
+  const classTypes = parseClasses.map(
```

Comment:
can you filtered out the ones with `!parseGraphQLSchema.parseClassTypes[parseClass.className].classGraphQLOutputType`?
It can happen due to name collision